gnssanalysis/_version.py

```diff
@@ -458,7 +483,7 @@ def render_pep440_pre(pieces):
     return rendered
 
 
-def render_pep440_post(pieces):
+def render_pep440_post(pieces: Dict[str, Any]) -> str:
     """TAG[.postDISTANCE[.dev0]+gHEX] .
```

Comment:
Might be good to start with a description and then this
Also, One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: `:param` `:return`)

Reply:
file autogenerated by the versionner script. so won't be addressed